### PR TITLE
created Debug Section

### PR DESCRIPTION
**Pull request by XI4507**

### CODE REVIEW

File: packages/alea-frontend/pages/debug-section.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          label="File Path"
+          variant="outlined"
+          value={path}
+          onChange={(e) => setPath(e.target.value)}
+        />
+        <Button variant="contained" onClick={() => fetchData(archive, path)}>
+          Search
+        </Button>
+      </Box>
+      <Box padding="10px">
```
Comment (abchugh) [inline packages/alea-frontend/pages/debug-section.tsx:42]: (nit): padding -> p

File: packages/alea-frontend/pages/debug-section.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        {sectionData != null ? (
+          <FileSectionDisplay sectionData={sectionData} />
+        ) : null}
```
Comment (abchugh) [inline packages/alea-frontend/pages/debug-section.tsx:45]: (nit): Can shorten to:
`sectionData && <FileSectionDisplay sectionData={sectionData} />`

File: packages/alea-frontend/components/FileSectionDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+export const FileSectionDisplay = ({
+  sectionData,
+}: {
+  sectionData: SectionsAPIData;
+}) => {
+  if (!sectionData) {
+    return null;
+  }
+  const { archive, filepath, title,children} = sectionData;
+  const isFile = sectionData.filepath && sectionData.archive;
```
Comment (abchugh) [inline packages/alea-frontend/components/FileSectionDisplay.tsx:17]: (nit): can simplify to `const isFile = filepath && archive;`

**Review by abchugh — APPROVED** [review]

**Comment by XI4507** [comment]
done the necessary changes

**Review by abchugh — APPROVED** [review]

**Pull request merged**